packages/components/src/components/DistributionChart.tsx

```diff
@@ -46,9 +48,18 @@ export function makePlot(record: {
 }
 
 export const DistributionChart: React.FC<DistributionChartProps> = (props) => {
-  const { plot, height, showSummary, width, logX, actions = false } = props;
+  const {
+    plot,
+    height,
+    showSummary,
+    width,
+    logX,
+    actions = false,
+    xAxis = "number",
+  } = props;
+  console.log({ plot });
```

Comment:
Remove. If you haven't yet, do a thorough pass of the code before submitting, for things like this.